File: ProjetBDDA/src/main/RelDef.java
package main;

public class RelDef {
		
	private int fileId;
	private RelSchema relSchema;
	
	public RelDef (int fileId, RelSchema relSchema){
		this.fileId = fileId;
		this.relSchema = relSchema;
	}

	public int getFileId() { 
		return fileId;
	}
	
	public RelSchema getRelShema() {
		return relSchema;
	}
	
}

File: ProjetOld/Codes/RelDef.php
<?php

	class RelDef {
		
		private $_fildeId;
		private $_relSchema;

		public function __construct($fileId,$relSchema) {
			$this->setFileId($fileId);
			$this->setRelSchema($relSchema);
		}

		public function setFileId($fileId) { $this->_fildeId = $fileId; }
		public function setRelSchema($relShema) { $this->_relSchema = $relShema; }

		public function getFileId() { return($this->_fildeId); }
		public function getRelSchema() { return($this->_relSchema); }
		
	}
?>